src/workflows/airqo_etl_utils/satellilte_utils.py

```
@@ -0,0 +1,160 @@
+
+        train_data.drop(columns=["date", "ID"], axis=1, inplace=True)
+        validation_data.drop(columns=["date", "ID"], axis=1, inplace=True)
+        test_data.drop(columns=["date", "ID"], axis=1, inplace=True)
+
+        train_target, validation_target, test_target = (
+            train_data[target_col],
+            validation_data[target_col],
+            test_data[target_col],
+        )
+
+        sampler = optuna.samplers.TPESampler()
```

Ruff

Undefined name `optuna`

(F821)